tools/combine_json_outut.py

```diff
@@ -0,0 +1,61 @@
+# This tool combines all json files written by news-please in one new file, one line per
```

Comment:
this seems to be unrelated to the filtering mentioned in the PR's title. could you remove these additions?

Reply:
I have removed b6881834094402e9ce23a6687e225c51e70d9f14 from my branch, it looks ok now.